### PR TITLE
feat: web app for previewing svg markup returned by `BlissSVGBuilder`

### PR DESCRIPTION
**Pull request by klown**
This adds another web application to the `./apps` folder, specifically `./apps/svg-preview`.

The app can be run on localhost and is started by the existing `npm run serveAppsDemos` script.  If so launched, use this following URL to access it:  `http://localhost:5173/apps/svg-preview/svg-preview.html`

### CODE REVIEW

**Comment by klown** [comment]
@cindyli I have requested a review from you, but I think I should modify the main README and possibly add a README to the `svg-preview` folder in order to document how to use this app.  However, feel free to start reviewing the code while I work on the documentation.

File: apps/svg-preview/svg-preview.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  </style>
+</head>
+<body>
+  <script type="module">
+    import "./svg-preview.ts";
+  </script>
+  <h1>Bliss SVG Previewer</h1>
+  <p>
+    The encoding string entered below is a space separated sequence of BCI-AV
+    identfiers and separators such as "/" and ";".  The sequence can be as
```
Comment (cindyli) [inline apps/svg-preview/svg-preview.html:18]: Would be helpful to explain what "/" and ";" mean. Something like:

* In the format of [12335, "/", 8499], the Bliss character 12335 and the Bliss character of 8499 are displayed side by side
* In the format of [12335, ";", 8499], the Bliss character 12335 and the indicator 8499 are displayed in the way that the indicator 8499 is on top of the the charactor 12355

Comment (klown) [reply apps/svg-preview/svg-preview.html:18]: Good point.  However, in exploring the Bliss SVG builder for more information, I discovered that this string will work as well:  "HC8N:0,8;S4:2,10".  It is used by the BMW palette for the first part of "Abs. Time", the "Abs." part.

That means that the legitimate encodings can be complicated and use letters, colons, and commas as well as slash and semi-colon.  I have adjusted the code to take that all into account.

However, I can't find any documentation for the structure of these encodings.  Do you know if there is any?

Comment (cindyli) [reply apps/svg-preview/svg-preview.html:18]: I don't know about the documentation. Hannes will know better.

**Comment by cindyli** [comment]
@klown do you think if this pull request is still needed. Seems the SVG preview is already covered by [the pull request](https://github.com/inclusive-design/adaptive-palette/pull/25) that generates palette JSON files.

**Comment by klown** [comment]
> @klown do you think if this pull request is still needed. Seems the SVG preview is already covered by [the pull request](https://github.com/inclusive-design/adaptive-palette/pull/25) that generates palette JSON files.

No, not anymore @cindyli .  It was the inspiration for the SVG string inputs for the palette generator PR.

I am closing this one as overtaken by #25.

**Pull request closed without merging**